src/openai-auth.ts

```diff
@@ -125,15 +127,23 @@ export async function getOpenAIAuth({
  * recognizes it and blocks access.
  */
 export async function getBrowser(launchOptions?: PuppeteerLaunchOptions) {
-  const macChromePath =
-    '/Applications/Google Chrome.app/Contents/MacOS/Google Chrome'
+  let executablePath
+
+  if (/^win/i.test(osPlatform)) {
+    executablePath = ' C:\\ProgramFiles\\Google\Chrome\\Application\\chrome.exe'
```

Comment:
space before `C:\\` and there are a few places here with multiple `\\` when there should only be one